drupal/sites/default/boinc/themes/boinc/templates/node-forum.tpl.php

```diff
@@ -110,8 +110,8 @@
   <?php if ($unpublished): ?>
     <div class="unpublished"><?php print bts('Unpublished', array(), NULL, 'boinc:comment-action-links'); ?></div>
   <?php endif; ?>
-  
-  <?php 
+
+  <?php
```

Comment:
All of these whitespace changes should be in a separate "clean up" commit, so as not to detract from the purpose of this commit. (Though I'm not sure it's even necessary, as they are not fixing indentation / readability issues?)

Reply:
Yes, please 👍